src/main/java/org/apache/maven/plugins/assembly/mojos/AbstractAssemblyMojo.java

```diff
@@ -404,6 +404,32 @@
     @Parameter( defaultValue = "${project.build.outputTimestamp}" )
     private String outputTimestamp;
 
+    /**
+     * Override of user ID in archive type which can store it.
+     */
+    @Parameter( property = "assembly.overrideUid" )
```

Comment:
Maybe it is better to name it maven.assembly.overrideUid ?

Cc @rfscholte

Reply:
Based on https://codehaus-plexus.github.io/plexus-archiver/apidocs/index.html?org/codehaus/plexus/archiver/Archiver.html it makes sense.

Reply:
All the other properties are just `assembly.*`.

Reply:
Ah, another question: does it make sense to set this via the commandline (`-Dkey=value`). I don't think so, or in other words: just use `@Parameter`

Reply:
IMHO, it seems nice to be able to do it via command line.

Reply:
Is there really a case the you when you need to change it for just one build? To me you want a consistent output, so hence not expose the parameter not with a property.
I noticed that the previous developers were quite enthousiastic with providing properties, but nowadays we're looking more closely if it really makes sense.

Reply:
I have removed the property names.